src/main/java/scm/address/logic/commands/FindAndExportCommand.java

```diff
@@ -101,7 +108,20 @@ private Predicate<Person> createPredicateForFiltering(String tag, String name, S
         return predicate;
     }
 
-    private void exportData(List<Person> users, String filename) throws IOException {
+    private void exportData(List<Person> users, String filename, String fileFormat) throws IOException {
+        switch (fileFormat) {
+        case "json":
+            exportDataAsJson(users, filename);
+            break;
+        case "csv":
+            exportDataAsCsv(users, filename);
+            break;
+        default:
+            throw new IOException("Unsupported file format: " + fileFormat);
```

Comment:
`Unsupported file format` should be put in a constant instead, I think?

Reply:
I agree. Thanks!